app/views/users/show.html.haml

```diff
@@ -0,0 +1,28 @@
+%h1 Users
+
+%table.table-striped.table-bordered
+  %tr
+    %th Fullname
+    %th Actions
+  - @users.each do |user|
+    %tr
+      %td= user.fullname
+      %td
+        = link_to 'Trim', trim_user_path(user), :id => "#{user.id}", :class => 'trim', :data => {:url => '/trim'}
```

Comment:
I kinda hate that ajaxify thing because it adds hidden magic without any
code or markup.  It would be extremely easy to end up ajaxifying things
unintendedly.  Suggest you use a helper or something else.  About to be out
of data range for a week here, best of luck!
On May 1, 2015 12:37 PM, "Rebecca Holzschuh" <notifications@github.com>
wrote:

> In app/views/users/show.html.haml
> <https://github.com/payrollhero/bearded-shame/pull/11#discussion_r29527616>
> :
>
> > @@ -0,0 +1,28 @@
> > +%h1 Users
> > +
> > +%table.table-striped.table-bordered
> > +  %tr
> > +    %th Fullname
> > +    %th Actions
> > +  - @users.each do |user|
> > +    %tr
> > +      %td= user.fullname
> > +      %td
> > +        = link_to 'Trim', trim_user_path(user), :id => "#{user.id}", :class => 'trim', :data => {:url => '/trim'}
>
> Hi Bram,
>
> Does this still need the method: :put parameter since I'm using ajax and
> preventing the default anyway?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/payrollhero/bearded-shame/pull/11/files#r29527616>.
>